e2e_testing/torchscript/vision_models.py

```diff
@@ -12,36 +12,67 @@
 
 # ==============================================================================
 
+
 class ResNet18Module(torch.nn.Module):
     def __init__(self):
         super().__init__()
         # Reset seed to make model deterministic.
         torch.manual_seed(0)
         self.resnet = models.resnet18()
         self.train(False)
+
     @export
-    @annotate_args([
-        None,
-        ([-1, 3, -1, -1], torch.float32, True),
-    ])
+    @annotate_args(
```

Comment:
nit: the testing framework has been using the existing format for `annotate_args `. could you change it back for consistency?

Reply:
Yeah I'm not sure how this happened. Probably VSCode autoformatting.